docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -144,14 +184,20 @@ Standard TensorFlow\* operations:
 | Const | No |
 | Conv2D | No |
 | Conv2DBackpropInput | No |
+| Conv3D | No |
+| Conv3DBackpropInputV2 | No |
 | Cos | No |
 | Cosh | No |
 | CropAndResize | "method" = "bilinear" only |
+| CTCGreedyDecoder | Supported only with decoded indices output in a dense format |
+| CTCLoss | Supported only with decoded indices input in a dense format |
 | CumSum | No |
 | DepthToSpace| No |
 | DepthwiseConv2dNative| No |
+| Elu | No |
```

Comment:
please add `Einsum` - Supported only with equation that does not contain repeated labels within subscript

Reply:
Done.